src/main/java/eu/execom/pomodoroTeam/controllers/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,33 @@
+package eu.execom.pomodoroTeam.controllers;
+
+import org.jboss.logging.Logger;
```

Comment:
This is not a good logging class. You should use SLF4J, and you can achieve it just by adding a Lombok annotation `@Slf4j` on your class.